src/main/java/com/cse/helium/app/controllers/ActorsController.java

```diff
@@ -30,7 +30,7 @@
 
   private static final Logger logger =   LogManager.getLogger(ActorsController.class);
 
-  @Autowired ActorsDao actorsDao;
+  @Autowired ActorsDao dao;
```

Comment:
Do we need this object declared as a property of ActorsController since it exists in the super?

Reply:
Talked with Jyotsna, we're good on the needed change.